boards/ek-lm4f120xl/include/periph_conf.h

```diff
@@ -68,26 +68,25 @@ extern "C" {
 #define TIMER_1_IRQ_CHAN    Timer1A_IRQn
 /** @} */
 
-
 /**
  * @name    UART configuration
  * @{
  */
 static const uart_conf_t uart_config[] = {
     {
-        .dev               = 0,
-        .uart_txint_mode   = UART_TXINT_MODE_EOT,
-        .uart_fifo_tx      = UART_FIFO_TX4_8,
-        .uart_fifo_rx      = UART_FIFO_RX4_8,
-        .uart_irq_chan     = UART0_IRQn,
-        .gpio_sysctl       = SYSCTL_PERIPH_GPIOA,
-        .gpio_port         = GPIO_PORTA_BASE,
+        .dev = 0,
+        .uart_txint_mode = UART_TXINT_MODE_EOT,
+        .uart_fifo_tx = UART_FIFO_TX4_8,
```

Comment:
sorry, this was a misunderstanding. this and the next lines were just missing a space...